src/helpers/mailer.js

```diff
@@ -0,0 +1,45 @@
+import { config } from 'dotenv';
+import mailer from '@sendgrid/mail';
+
+config();
+
+const sendEmail = async (action, to, token) => {
+  const {
+    SENDGRID_API_KEY, EMAIL_SENDER, APP_URL, NODE_ENV
+  } = process.env;
+  mailer.setApiKey(SENDGRID_API_KEY);
+  const subjectVerify = 'Welcome to Authors Haven';
+  const contentVerify = '<h2>Congratulations on your new Authors Haven account!</h2><p style="font-size: 1rem;">Please confirm your email address by visiting the following link with <strong>Postman</strong>:</p>';
+  const linkVerify = `${APP_URL}/${token}`;
+
+  if (action === 'verify-email') {
+    const message = {
+      to,
+      from: EMAIL_SENDER,
+      subject: subjectVerify,
+      text: 'Authors Haven',
+      html: `<div style="background:#ECF0F1;width:100%;padding:20px 0;">
```

Comment:
I'd much rather this markup string is generated by a function e.g `emailTemplateGenerator(contentVerify, linkVerify)` and used here. Would make it much cleaner

Reply:
Thank you. I'm going to implement that approach. It is really good.